napari/conftest.py

```diff
@@ -21,9 +22,14 @@
 if not hasattr(pooch.utils, 'file_hash'):
     setattr(pooch.utils, 'file_hash', pooch.hashes.file_hash)
 
-try:
+
+SKIMAGE_VERSION = tuple(int(v) for v in skimage.__version__.split('.'))
+
+if SKIMAGE_VERSION[:2] >= (0, 19):
+    from skimage.data._fetchers import image_fetcher
+elif SKIMAGE_VERSION[:2] >= (0, 16):
     from skimage.data import image_fetcher
-except ImportError:
+else:
```

Comment:
For 0.19.1, we can make `image_fetcher` available under `skimage.data` again. This was an unintended change made when adding lazy-loading to this module (apparently we don't have any test cases that call `image_fetcher` directly).

Given that, I wouldn't recommend changing to import from the private `_fetchers` module going forward. 

Maybe we should just disallow 0.19.0 specifically in the napari requirements and then I will make sure the existing code here works for 0.19.1 before we tag it?

Reply:
> Maybe we should just disallow 0.19.0 specifically in the napari requirements and then I will make sure the existing code here works for 0.19.1 before we tag it?

fine by me - cc @jni. would have been great to test this in our "pre-" tests on main and surface before release, but i don't think we check them enough!

Reply:
Disallowing a version would definitely be the best in that case! How should we go about this for now?

Reply:
this has been done by #3749